src/style.css

```diff
@@ -1,5 +1,28 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+* {
+  font-family: 'Lato', sans-serif;
+  color: #0e0439;
+}
+
+body {
+  height: 95vh;
+  background: rgb(1, 0, 5);
+  background: linear-gradient(45deg, rgba(1, 0, 5, 1) 0%, rgb(14, 4, 57) 100%);
+}
+
+h1 {
+  color: white;
+  padding-left: 20px;
+  font-size: 30px;
+}
```

Comment:
- [ ] Please style the 'Leaderboard' heading text so that it looks like the wireframe. Kindly increase the `font-size` and align it closely to the left as shown below 👇 
- [ ] Please also note that the submit button should be aligned to the right and not take up all the available space of its container like you have done
### Wireframe
![image](https://user-images.githubusercontent.com/46839548/233514241-68fc9604-0313-4818-97ed-354e9b81b4d0.png)

### Your Design
![image](https://user-images.githubusercontent.com/46839548/233514729-46acfbe3-b1f6-4654-92ca-3f0ece19a684.png)

Reply:
Hi @codepantha, thank you for your required changes.